README.md

```diff
@@ -106,111 +107,89 @@ conda config --set solver libmamba
 
 ### Windows:
 
-Try:
-
-```
-conda env create --file .\install\gym.yml
-```
-
-If the above fails, and 
 
+1) you wish to use GPU for model training and the latest Tensorflow version, you now must use WSL2 and refer to the [official Tensorflow instructions](https://www.tensorflow.org/install/pip#windows-native_1). These instructions are therefore catered to WSL2 users.
```

Comment:
It might be helpful to just directly tell the user  these instructions are ONLY for wsl2 because this will not work on Windows native